blocks/v2-feature-carousel/v2-feature-carousel.js

```diff
@@ -0,0 +1,95 @@
+import {
+  createElement,
+  unwrapDivs,
+  adjustPretitle,
+} from '../../scripts/common.js';
+import {
+  listenScroll,
+  createArrowControls,
+} from '../../scripts/carousel-helper.js';
+
+const blockName = 'v2-feature-carousel';
+
+const updateActiveClass = (elements, entry) => {
+  elements.forEach((el, index) => {
+    if (el === entry.target) {
+      el.classList.add('active');
+      let arrowControl = document.querySelector(`.${blockName}__button:disabled`);
+
+      if (arrowControl) {
+        arrowControl.disabled = false;
+        arrowControl = null;
+      }
+      // disable arrow buttons
+      if (index === 0) {
+        arrowControl = document.querySelector(`.${blockName}__button-prev`);
+      } else if (index === el.parentNode.children.length - 1) {
+        arrowControl = document.querySelector(`.${blockName}__button-next`);
+      }
+      if (arrowControl) {
+        arrowControl.disabled = true;
+      }
+    } else {
+      el.classList.remove('active');
+    }
+  });
+};
+
+const arrowFragment = document.createRange().createContextualFragment(`<li>
```

Comment:
This variable is shared by many instances of the `v2-feature-carousel`, it will not work if there is more than one `v2-feature-carousel` on the same page. You can check it here: https://39-feature-carousel--vg-macktrucks-com-rd--netcentric.hlx.page/drafts/tdziezyk/v2/test
The second carousel doesn't have the arrows